### PR TITLE
Add skins export

### PR DESCRIPTION
**Pull request by samuveth**
For https://github.com/snapshot-labs/snapshot/pull/499

### CODE REVIEW

File: skins.js [hunk 1]
```diff
@@ -0,0 +1,415 @@
+import aave from "./skins/aave.scss";
```
Comment (bonustrack) [inline skins.js:1]: This should be enough:
```ts
import './skins/aave.scss';
import './skins/aavegotchi.scss';
...
```
Also we don't need export

Comment (bonustrack) [reply skins.js:1]: Also can you change the filename to `./skins/index.ts` ?

Comment (samuveth) [reply skins.js:1]: How would be import skins here then?

Comment (bonustrack) [reply skins.js:1]: @samuveth Oh right, sorry forgot about that, then you can keep the way is it, just change " with ' and the filename to skins/index.ts

**Review by bonustrack — APPROVED** [review]
Looks good!

**Comment by bonustrack** [comment]
I'll just change " to ' and filename to index.ts

**Comment by samuveth** [comment]
> I'll just change " to ' and filename to index.ts

`.ts` doesn't work for me, if get this `Cannot find module './aave.scss' or its corresponding type declarations.ts(2307)`

**Comment by bonustrack** [comment]
@samuveth Where you get this error? On snapshot UI or this repo?

**Comment by samuveth** [comment]
> @samuveth Where you get this error? On snapshot UI or this repo?

@bonustrack On the Ui.
image

**Comment by bonustrack** [comment]
@samuveth Ok weird, can change to js then

**Pull request merged**